circleci/catapult-publish

```diff
@@ -38,11 +40,42 @@ BUILD_NUM=$CIRCLE_BUILD_NUM
 : ${CIRCLE_BRANCH?"Missing required env var"}
 BRANCH=$CIRCLE_BRANCH
 : ${CIRCLE_PROJECT_REPONAME?"Missing required env var"}
-REPO=$CIRCLE_PROJECT_REPONAME
+REPO_NAME=$CIRCLE_PROJECT_REPONAME
 : ${CIRCLE_SHA1?"Missing required env var"}
 SHORT_SHA=${CIRCLE_SHA1:0:7}
 FULL_SHA=${CIRCLE_SHA1}
 
+
+. $DIR/install-goci
+
+set +e
+echo "Running goci."
+# Run the validate step using goci and capture the output
+VALIDATE_OUTPUT=$(goci validate)
+GOCI_EXIT_CODE=$?
+
+# If goci fails, we exit with the goci exit code and comment on the PR with the goci output
+if [[ $GOCI_EXIT_CODE -ne 0 ]]; then
```

Comment:
IMO, only a failed validation check should post a comment. If we exit code (1) due to a transient issue then we should failed CI normally without a PR comment.